osu.Game.Rulesets.Taiko/Beatmaps/TaikoBeatmapConverter.cs

```diff
@@ -182,7 +182,15 @@ private bool shouldConvertSliderToHits(HitObject obj, IBeatmap beatmap, IHasDist
 
             // The true distance, accounting for any repeats. This ends up being the drum roll distance later
             int spans = (obj as IHasRepeats)?.SpanCount() ?? 1;
-            double distance = distanceData.Distance * spans * LegacyBeatmapEncoder.LEGACY_TAIKO_VELOCITY_MULTIPLIER;
+
+            double distance;
+
+            if (obj is IHasPath pathData)
+                distance = pathData.Path.ExpectedDistance.Value ?? 0;
+            else
+                distance = distanceData.Distance;
```

Comment:
Does this fallback case realistically get hit under some scenario?

Reply:
As far as I can tell, no. I only added it for sanity/safety.